CHANGELOG.md

```diff
@@ -12,7 +12,8 @@ BREAKING_CHANGES:
 * Peering:
   * Remove support for customizing the server addresses in peering token generation. Instead, mesh gateways should be used
     to establish peering connections if the server pods are not directly reachable. [[GH-1610](https://github.com/hashicorp/consul-k8s/pull/1610)]
-  * Enabling peering requires `tls.enabled`. [[GH-1610](https://github.com/hashicorp/consul-k8s/pull/1610)]
+  * Enabling peering requires `global.tls.enabled`. [[GH-1610](https://github.com/hashicorp/consul-k8s/pull/1610)]
+  * Enabling peering requires `meshGateway.enabled`. [[GH-1683](https://github.com/hashicorp/consul-k8s/pull/1683)]
```

Comment:
nit: could we rephrase to be "verb ..." so "Require `global.tls.enabled` when peering is enabled" 🙏